RPIPs/RPIP-62.md

```diff
@@ -8,27 +8,27 @@ status: Draft
 type: Protocol
 category: Core
 created: 2024-07-25
+requires: rpip-63
```

Comment:
```suggestion
requires: 63
```

Reply:
I think this should just be the RPIP number for the website to generate a correct link.